packages/react-native-reanimated/src/hook/commonTypes.ts

```diff
@@ -13,13 +13,14 @@ import type {
   ViewStyle,
   NativeScrollEvent,
 } from 'react-native';
-import type { ViewDescriptorsSet, ViewRefSet } from '../ViewDescriptorsSet';
+import type { ViewDescriptorsSet } from '../ViewDescriptorsSet';
 import type { AnimatedStyle } from '../helperTypes';
+import { ReanimatedHTMLElement } from '../js-reanimated';
 
 export type DependencyList = Array<unknown> | undefined;
 
 export interface Descriptor {
-  tag: number;
+  tag: number | ReanimatedHTMLElement;
```

Comment:
I'm considering whether we should change the name of this prop from `tag` to something more accurate, but I'm unsure if it's a good idea. If anyone has a strong opinion, I'm open to suggestions.

Reply:
We can do that in a separate PR.

Reply:
Imho `tag` is fine, the only thing that has changed is that now it may represent `HTMLElement`. On the other hand, those elements are also called `tags` so we could leave that.

But if you have en example of more descriptive name, we can rename it.